backend/addcorpus/models.py

```diff
@@ -448,11 +448,12 @@ def clean(self):
                     e
                 ])
 
+
 class CorpusDocumentationPage(models.Model):
     class PageType(models.TextChoices):
         GENERAL = ('general', 'General information')
         CITATION = ('citation', 'Citation')
-        LICENSE = ('license', 'Licence')
+        LICENSE = ('license', 'License')
```

Comment:
This is a difference between British and American spelling; in American spelling, ["license"](https://www.merriam-webster.com/dictionary/license) is both a noun and a verb. In UK spelling, ["license"](https://dictionary.cambridge.org/dictionary/english/license) is the verb, and ["licence"](https://dictionary.cambridge.org/dictionary/english/licence) the noun.

Since I-analyzer normally follows British spelling, I had used that here.